apps/condo/domains/miniapp/schema/AppMessageSetting.test.js

```diff
@@ -0,0 +1,429 @@
+/**
+ * Generated by `createschema miniapp.AppMessageSetting 'app?:Relationship:B2CApp:CASCADE; blockReason:Text; blockedNotificationTypes:Text; throttlingTime:Integer;'`
+ */
+
+const {
+    makeLoggedInAdminClient, makeClient, expectValuesOfCommonFields,
+    expectToThrowUniqueConstraintViolationError, expectToThrowGQLError, catchErrorFrom,
+} = require('@open-condo/keystone/test.utils')
+const {
+    expectToThrowAuthenticationErrorToObj, expectToThrowAuthenticationErrorToObjects,
+    expectToThrowAccessDeniedErrorToObj,
+} = require('@open-condo/keystone/test.utils')
+
+const { CONTEXT_FINISHED_STATUS } = require('@condo/domains/miniapp/constants')
+const { AppMessageSetting, createTestAppMessageSetting, updateTestAppMessageSetting,
+    createTestB2CApp, createTestB2BApp, createTestB2BAppContext } = require('@condo/domains/miniapp/utils/testSchema')
+const { B2C_APP_MESSAGE_PUSH_TYPE, B2B_APP_MESSAGE_PUSH_TYPE, VOIP_INCOMING_CALL_MESSAGE_TYPE, TICKET_CREATED_TYPE } = require('@condo/domains/notification/constants/constants')
+const { createTestOrganization, createTestOrganizationEmployeeRole, createTestOrganizationEmployee } = require('@condo/domains/organization/utils/testSchema')
+const { makeClientWithNewRegisteredAndLoggedInUser, makeClientWithSupportUser } = require('@condo/domains/user/utils/testSchema')
+
+const { ERRORS } = require('./AppMessageSetting')
+
+
+describe('AppMessageSetting', () => {
+    let admin, support, anonymous, user, b2cApp, b2bApp
+
+    beforeAll(async () => {
+        admin = await makeLoggedInAdminClient()
+        support = await makeClientWithSupportUser()
+        user = await makeClientWithNewRegisteredAndLoggedInUser()
+        anonymous = await makeClient()
+    })
+
+    beforeEach(async () => {
+        const [testB2cApp] = await createTestB2CApp(admin)
+        b2cApp = testB2cApp
+        const [testB2bApp] = await createTestB2BApp(admin)
```

Comment:
A small change that makes tests 1 step closer to production
```suggestion
        const [testB2cApp] = await createTestB2CApp(support)
        b2cApp = testB2cApp
        const [testB2bApp] = await createTestB2BApp(support)
```